workflows/src/main/kotlin/com/template/flows/ApproveAppointmentRequest.kt

```diff
@@ -62,28 +72,20 @@ class ApproveAppointmentRequest(private val alice: Party,
 
         // Step 6. Collect the other party's signature using the SignTransactionFlow.
         val otherParties: MutableList<Party> = output.participants.stream().map { el: AbstractParty? -> el as Party? }.collect(Collectors.toList())
```

Comment:
Did you consider using mutableListOf here? https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/mutable-list-of.html